packages/dotcom/package.json

```diff
@@ -28,7 +28,7 @@
         "rollup-plugin-external-globals": "^0.5.0",
         "rollup-plugin-filesize": "^9.0.2",
         "rollup-plugin-peer-deps-external": "^2.2.3",
-        "rollup-plugin-terser": "^6.1.0",
+        "rollup-plugin-terser": "^7.0.2",
```

Comment:
rollup-plugin-terser is deprecated and should be replaced by `[@rollup/plugin-terser](https://www.npmjs.com/package/@rollup/plugin-terser#rollupplugin-terser)` - will also mean a small change to the rollup file to fix the import. Probably best to discuss with the DCR guardians and do the work in a separate PR?

Reply:
👍 good spot. There are probably a few deps that can be bumped here, but perhaps it's for a follow up PR